lms/djangoapps/certificates/api.py

```diff
@@ -228,8 +229,13 @@ def certificate_downloadable_status(student, course_key):
         'is_unverified': True if current_status['status'] == CertificateStatuses.unverified else False,
         'download_url': None,
         'uuid': None,
+        'is_invalid_cert': False,
```

Comment:
I don't believe putting this `is_invalid` check here is correct.
There is another function within this file called `cert_generation_enabled`. Why don't we put the check on CertInvalidation there?